packages/starlight/utils/navigation.ts

```diff
@@ -228,6 +228,9 @@ function treeify(routes: Route[], baseDir: string): Dir {
 		// Build the tree
 		.forEach((doc) => {
 			const parts = getBreadcrumbs(doc.id, baseDir);
+			// If the breadcrumb is empty, the document is at the root level so we can fallback to the
+			// document ID.
+			if (parts.length === 0) parts.push(doc.id);
```

Comment:
Just to make sure I understand: is this because previously `index.md` could be an ID and this would no longer generate any breadcrumbs because it has a slug of `"/"` or `""` or something? I’d like to understand why this handling is different/if it should be part of `getBreadcrumbs()`.